### PR TITLE
feat: adding missing fields to entities PAGOPA-1167

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Adding new fields and modifying existing ones, according to the description of the task [https://pagopa.atlassian.net/jira/software/c/projects/PAGOPA/boards/336?modal=detail&selectedIssue=PAGOPA-1167&assignee=6315beb0b433b060db56799b](url)

#### Motivation and Context
These changes are required to correctly map the biz event datastore model to the format of the positive business event produced by NDP

#### How Has This Been Tested?
This addition has been tested manually 

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FedericoRuzzier** [comment]
This PR has to wait, the reason is linked [here](https://pagopa.atlassian.net/jira/software/c/projects/PAGOPA/boards/336?modal=detail&selectedIssue=PAGOPA-1167&assignee=6315beb0b433b060db56799b)

File: src/main/java/it/gov/pagopa/bizeventsdatastore/BizEventEnrichment.java [hunk 1]
```diff
@@ -63,6 +63,7 @@ public void processBizEventEnrichment(
 		Logger logger = context.getLogger();
 
 		String msg = String.format("BizEventEnrichment stat %s function - num events triggered %d", context.getInvocationId(),  items.size());
+		logger.info("test");
```
Comment (pasqualespica) [inline src/main/java/it/gov/pagopa/bizeventsdatastore/BizEventEnrichment.java:66]: useless log , right ?

**Review by pasqualespica — APPROVED** [review]
lgtm

**Comment by pasqualespica** [comment]
@aferracci i closed this PR.
Let's first close the detailed documentation on biz+ and then evaluate any structural alignments

**Pull request closed without merging**